index.js

```diff
@@ -372,20 +372,18 @@ export default class WebTorrent extends EventEmitter {
    * @param  {string|Buffer|Torrent}   torrentId
    * @param  {function} cb
    */
-  remove (torrentId, opts, cb) {
+  async remove (torrentId, opts, cb) {
```

Comment:
hate this mix of async and cb

Reply:
this was to keep functionality mostly identical, IMHO the migration from cb -> async should all happen in a separate PR, not some version bump, I'd like to keep it like this for now and address it in a future PR, is that okay or should I remove cb in this one? [imho we really shouldnt]